redfish.conf

```diff
@@ -14,9 +14,18 @@ userName = administrator
 password =
 
 [ssl]
-SSLType = adhoc
-SSLCertFile =
-SSLKeyFile =
+SSLType = self-signed
+SSLCertFile = certs/self-signed.crt
+SSLKeyFile = certs/self-signed.key
+
+[ssl-cert-defaults]
```

Comment:
@danielreed @bsoutham: Does HPE OneView related stuff have default values to self-signed certificates?

Reply:
@ricardoas Fixed with values informed by Blaine.

Reply:
thanks!